.circleci/config.yml

```diff
@@ -59,7 +59,9 @@ jobs:
       - MATPLOTLIB_VERSION: 'latest'
       - CYTHON_VERSION: 'latest'
       - SCIKIT_IMAGE_VERSION: 'latest'
-      - SPHINX_VERSION: 'min'
+      # Bump the sphinx version from time to time. Avoid latest sphinx version
+      # that tends to break things slightly too often
+      - SPHINX_VERSION: 4.2.0
```

Comment:
I always find it amusing that a change which is already on `main` shows up in a PR

Reply:
Yes, it looks like it missed up with commits.